### DISCUSSION TITLE
Multiple questions

### DISCUSSION THREAD

**Discussion by sudobrat** [prasanthrangan/hyprdots#100] [Q&A] [answered]
1. How do I see all keybindings?
2. Some apps like vscode are very enlarged how to fix that? 
3.how to set up natural scrolling? 

Thanks in advance 🙂

**Comment by jhenrique04** [comment, marked as answer]
1) You can see all your keybindings in .config/hypr/keybindings.conf
2) I actlly had that same issue while using yay for some reason, however this did the trick for me:

creating a `code-flags.conf` in .config and putting `--enable-features=UseOzonePlatform,WaylandWindowDecorations --ozone-platform=wayland inside`. 

If it doesn't work, see this link: https://www.reddit.com/r/hyprland/comments/15jb7l7/blurry_vscode_on_hyprland/ , there are other ways of trying to fix it.

3) Open .config/hypr/hyprland.conf, scroll to the "input" tab
![image](https://github.com/prasanthrangan/hyprdots/assets/115225048/baa27ec7-31c5-416b-b53a-476d5253abaa)
Where it says natural_scroll should be false, you just need to change it to true.

**Reply by sudobrat** [reply]
Thanks a lot 🙂 loved this setup btw

**Comment by jhenrique04** [comment]
Your welcome

I liked it too, really is the best for me
